slashing-protection/referencetests/build.gradle

```diff
@@ -29,6 +29,7 @@ dependencies {
   testImplementation 'com.fasterxml.jackson.core:jackson-databind'
   testImplementation 'org.apache.logging.log4j:log4j-api'
   testImplementation 'org.apache.logging.log4j:log4j-core'
+  testImplementation 'com.google.guava:guava'
```

Comment:
why is this extra dependency needed now?

Reply:
This was previously resolved via transitive dependency for slashing-protection/referencetests module. After changing tuweni version, the Google common `Resources.getResource...` call wasn't resolving it within this module anymore.